sphinx/source/docs/user_guide/server.rst

```diff
@@ -1430,55 +1408,57 @@ Then in your web application, we explicitly provide (signed) session ids using
                             session_id=generate_session_id())
     return render_template("embed.html", script=script, template="Flask")
 
-Make sure that the ``BOKEH_SECRET_KEY`` environment variable is set (and
-identical) for both the Bokeh server and web app processes (e.g. Flask or
-Django or whatever tool is in use).
+Make sure to set identical ``BOKEH_SECRET_KEY`` environment variables both for
+the Bokeh server and for the web app processes, be it Flask, Django, or any
```

Comment:
"be it ..." rings a bit unusual to me. Maybe just just delete the last part after "web app process" entirely?